### PR TITLE
add mlu stack kernel

### PR DESCRIPTION
**Pull request by cifar10**
### PR types
New features

### PR changes
OPs

### Describe
add stack forward

### CODE REVIEW

**Comment by cifar10** [comment]
stack

File: paddle/fluid/operators/stack_op_mlu.cc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    PADDLE_ENFORCE_GT(
+        num, 0,
+        platform::errors::InvalidArgument("number of input Tensor <= 0"));
+
+    std::vector<MLUCnnlTensorDesc> x_descs;
+    std::vector<cnnlTensorDescriptor_t> x_raw_descs;
+    std::vector<const void*> x_ptrs;
+    for (int i = 0; i < num; i++) {
+      if (x[i]->dims().size() != 0) {
```
Comment (ShawnNew) [inline paddle/fluid/operators/stack_op_mlu.cc:42]: 感觉这块逻辑只需要把所有input的dims组成vector就好？

**Review by ShawnNew — CHANGES_REQUESTED** [review]

**Review by ShawnNew — APPROVED** [review]
👍

**Comment by Ivorfeng** [comment]
👍

**Review by qili93 — APPROVED** [review]
LGTM

**Pull request merged**